lib/index.js

```diff
@@ -12,56 +12,51 @@ const IdGenerator = function(prefixes){
   else if (typeof prefixes === 'object') {
     this.prefix = prefixes.prefix;
     this.prefixes = [this.prefix]
-    this.len = prefixes.len  || 16;
+    this.len = prefixes.len || 16;
     this.alphabet = prefixes.alphabet || ALPHA_NUM;
     this.separator = prefixes.separator || '_';
   }
   else {
     this.len = 16;
     this.alphabet = ALPHA_NUM;
   }
-  this.prefixes = this.prefixes ||  [];
+  this.prefixes = this.prefixes || [];
 };
 
-IdGenerator.prototype.get = function() {
-  const rnd = crypto.randomBytes(this.len);
+IdGenerator.prototype.get = function () {
   const value = new Array(this.len);
   const charsLength = this.alphabet.length;
 
   for (var i = 0; i < this.len; i++) {
-    value[i] = this.alphabet[rnd[i] % charsLength];
+    value[i] = this.alphabet[randomInteger(charsLength - 1)];
   }
 
-  if(this.prefix) {
+  if (this.prefix) {
     return this.prefix + this.separator + value.join('');
   }
   else {
     return value.join('');
   }
 };
 
-IdGenerator.prototype.newUid = function(len) {
-  const rnd = crypto.randomBytes(len);
+IdGenerator.prototype.newUid = function (len) {
   const value = new Array(len);
-  const charsLength = ALPHA_NUM.length;
-
   for (var i = 0; i < len; i++) {
-    value[i] = ALPHA_NUM[rnd[i] % charsLength];
+    value[i] = ALPHA_NUM[randomInteger(ALPHA_NUM.length - 1)];
```

Comment:
If `maximum` is exclusive should this not be `ALPHA_NUM[randomInteger(ALPHA_NUM.length)]`?